main.go

```diff
@@ -42,7 +31,7 @@ func main() {
 	flagSet.String("tls-key-file", "", "path to private key file")
 	flagSet.String("redirect-url", "", "the OAuth Redirect URL. ie: \"https://internalapp.yourcompany.com/oauth2/callback\"")
 	flagSet.Bool("set-xauthrequest", false, "set X-Auth-Request-User and X-Auth-Request-Email response headers (useful in Nginx auth_request mode)")
-	flagSet.Var(&upstreams, "upstream", "the http url(s) of the upstream endpoint, file:// paths for static files or static://<status_code> for static response. Routing is based on the path")
+	flagSet.StringSlice("upstream", []string{}, "the http url(s) of the upstream endpoint, file:// paths for static files or static://<status_code> for static response. Routing is based on the path")
```

Comment:
I think pflag`FlagSet.StringArray` has behavior closer to the previous internal `StringArray`, whereas this `FlagSet.StringSlice` splits values on comma (in addition to taking multiple flags/values).

Reply:
I think the old behaviour was closer to `StringArray` (based off this [description](https://github.com/spf13/pflag/issues/89)). But, at least for upstream where you can't have a comma in a url then using `StringSlice` would be better.